src/Payum/Paypal/ExpressCheckout/Nvp/Api.php

```diff
@@ -426,6 +426,24 @@ public function doExpressCheckoutPayment(array $fields)
      *
      * @return array
      */
+    public function cancelRecurringPaymentsProfile(array $fields)
+    {
+        $request = new FormRequest();
+        $request->setFields($fields);
+
+        $request->setField('METHOD', 'CancelRecurringPaymentsProfile');
```

Comment:
Oops, indeed it doesn't exist. I misread the docs. This should be a call to `ManageRecurringPaymentsProfileStatus` with 2 arguments, `PROFILEID` and `ACTION`. Let me make a patch (probably this afternoon)